src/index.ts

```diff
@@ -34,73 +38,30 @@ export async function execute(filenames: string[], options: Partial<ExecutionOpt
     const executionOptions = Object.assign(defaultExecutionOptions, options);
 
     let manifest = await PP.getManifest(executionOptions.root);
+    let componentDefinitions = await getComponentDefinitionMap(executionOptions.root);
+
+    componentDefinitions.forEach((component: ComponentDefinition) => {
+        if (component.scripts.length < 1) return;
```

Comment:
So... this one is a weird situation that probably needs to be refactored some but I didn't want to work on it immediately. If you think it's worth fixing now I can go back and try and rework it. Otherwise we can note it and come back later.

Basically we have a mutation going on here for each component. If there are scripts present we need to modify the current component's set of paths to be correct. If it doesn't have any scripts then we just want to skip over it. I don't really like the fact that we're mutating the paths here but it works for this situation.